README.markdown

```diff
@@ -1,144 +1,180 @@
-This is the [Uploadcare](https://uploadcare.com/) widget source.
+# Uploadcare Widget
 
-[![Build Status](https://travis-ci.org/uploadcare/uploadcare-widget.png?branch=master)](https://travis-ci.org/uploadcare/uploadcare-widget)
+<a href="https://uploadcare.com/?utm_source=github&utm_campaign=uploadcare-widget">
+    <img align="right" width="64" height="64"
+         src="https://ucarecdn.com/2f4864b7-ed0e-4411-965b-8148623aa680/uploadcare-logo-mark.svg"
+         alt="">
+</a>
 
-## Embedding
+Uploadcare Widget is an HTML5 file uploader, a part of the
+[Uploadcare](https://uploadcare.com/?utm_source=github&utm_campaign=uploadcare-widget)
+ecosystem that fully covers your file handling.
 
-Add following code to your document's `<head>`.
+---
 
-```html
-<script>
-  // Widget settings
-  UPLOADCARE_PUBLIC_KEY = 'your_public_key';
-</script>
-<script src="https://ucarecdn.com/widget/x.y.z/uploadcare/uploadcare.full.min.js" charset="utf-8"></script>
-```
+**Note: we have a new major version of the widget. [Here’s how][widget-docs-migration-v2-v3] you migrate from v2 to v3.**
```

Comment:
Note: there currently is a new major version of the widget. Here's how you migrate from v2 to v3.